lift.css

```diff
@@ -0,0 +1,353 @@
+:root{
+    --primary-color: #cfeefb;
+    --grey-color: gray;
+    --green-success:  rgb(27, 170, 27);
+    --red-failed:rgb(253, 37, 37);
+    --yellow-warning:#ff9800;
+
+}
+
+body{
+    font-family: 'Montserrat', sans-serif;
+    margin: 0px
+}
+
+.index-card{
+    width: 100%;
+    height: 100%;
+    margin:10px;
+    padding: 2rem;
+    text-align: center;
+}
```

Comment:
code repetition alert
text-align: center (repeated 10 times) use CSS variable as u used for the colors.